packages/core/src/entity/order-item/order-item.entity.ts

```diff
@@ -27,6 +27,9 @@ export class OrderItem extends VendureEntity {
     @ManyToOne(type => OrderLine, line => line.items, { onDelete: 'CASCADE' })
     line: OrderLine;
 
+    @EntityId()
+    lineId: ID;
```

Comment:
Where is this used?

Reply:
This was the main key change needed next to the `.execute()`: without it the `.insert()` call ignored the foreign key `lineId` when bulk inserting the order items (a bit strange, had a look in the TypeORM insert query builder to understand it). Resulting in `null` columns in the database.

I however changed the `OrderModifier` code to use `lineId: orderLine.id` instead of `line: orderLine` to be more clear. Awaiting the e2e tests to see if this still behaves the same. Otherwise, I'll revert it and add a comment to the `@EntityId` line.